packages/storybook-builder/package.json

```diff
@@ -70,9 +71,10 @@
     "remark-slug": "^6.0.0",
     "rollup": "^4.4.1",
     "rollup-plugin-external-globals": "^0.9.0",
-    "slash": "^5.1.0"
+    "slash": "^5.1.0",
+    "storybook": "^8.3.3"
```

Comment:
I don't know much about this project, but in 99 % of the cases, `storybook` should be a peer dependency here, because projects should always have that installed explicitly.

Maybe that's not the case for your setup here though.

Reply:
I think I was trying to obtain channels and forgot to uninstall storybook, which depends on `@storybook/core`. I'll remove it